lib/ctcs2_to_junit.pm

```diff
@@ -71,8 +71,8 @@ sub generateXML {
         failures  => "$fail_num",
         hostname  => `hostname`,
         id        => "0",
-        name      => get_var("QA_TESTSUITE"),
-        package   => get_var("QA_TESTSUITE"),
+        name      => get_var('QA_TESTSET') || "xfstests",
```

Comment:
you may stay with your version , but just FYI that you can also use second parameter of `get_var` function : 
```suggestion
        name      => get_var('QA_TESTSET',"xfstests"),
```

Reply:
after reading code further , just to make sure that you haven't confuse yourself here . Maybe you actually wanted to write : 
```suggestion
        name      => get_var('QA_TESTSET') || get_var("XFSTESTS"),
```

Reply:
generally it looks fishy - you have "QA_TESTSET" variable which suppose to handle test suite name ( as I understand at least ) and you have some special test suite which not represented via this variable but has different Boolean variable as indicator , but it is far beyond this PR . just general thoughts ...

Reply:
Thanks, I'd like to fix this.